### DISCUSSION TITLE
Prevent Pylance from overriding git's highlights

### DISCUSSION THREAD

**Discussion by lucasrabiec** [microsoft/pylance-release#3199] [Q&A]
Is there any way to prevent Paylance from overriding git's highlights of modified files in the project tree view?
![Screenshot 2022-08-16 at 13 04 23](https://user-images.githubusercontent.com/9741188/184865197-038e94f2-27d1-4e8c-82df-462783480c1d.png)

Pylance version: v2022.8.20

**Comment by luabud** [comment]
Hey @lucasrabiec! I don't think Pylance overrides git's information, the number of problems should show next to the modified marker that comes from the git functionality in VS Code. Like this:

![image](https://user-images.githubusercontent.com/45497113/184945302-3a496798-5532-4f36-924e-a4449c13116b.png)

I'm wondering if you may have a setting or another extension that may be causing this?

**Reply by lucasrabiec** [reply]
When I disabled Pylance, git's highlights worked well. Today when I opened VS Code the problem was gone and now it looks like on your ss. Weird because I restarted VS Code a few times yesterday.

**Comment by bschnurr** [comment]
maybe disabling this helps. 
![image](https://user-images.githubusercontent.com/1946977/184946827-246fb9ca-6ae5-470f-9cf4-48e314171898.png)

**Reply by lucasrabiec** [reply]
Thanks, it can be some kind of tradeoff when the problem occurs again.
